.eslintrc.json

```diff
@@ -85,10 +85,9 @@
       "googlesitekit-modules",
       "googlesitekit-api",
       "googlesitekit-widgets",
-      "googlesitekit-components",
       "@wordpress/i18n__non-shim",
       "@wordpress/api-fetch__non-shim",
-      "@wordpress/element__non-shim"
+      "googlesitekit-components"
```

Comment:
Why did we move the `googlesitekit-components` entry as part of this change?

Reply:
The ordering was originally like this. Since I used the revert functionality in GitHub to revert the PR (and then rebased `main` into it), the ordering was reverted as well. I added [a new commit](https://github.com/google/site-kit-wp/pull/6111/commits/4ad126affc873c93bf6a44d5723de5a9b72126e6) to preserve the ordering as it is better this way.